src/urllib3/contrib/_appengine_environ.py

```diff
@@ -6,24 +6,20 @@
 
 
 def is_appengine():
-    return (is_local_appengine() or
-            is_prod_appengine() or
-            is_prod_appengine_mvms())
+    return 'APPENGINE_RUNTIME' in os.environ
 
 
 def is_appengine_sandbox():
     return is_appengine() and not is_prod_appengine_mvms()
 
 
 def is_local_appengine():
-    return ('APPENGINE_RUNTIME' in os.environ and
-            'Development/' in os.environ['SERVER_SOFTWARE'])
+    return ('SERVER_SOFTWARE' not in os.environ or
+            os.environ['SERVER_SOFTWARE'].startswith('Development'))
 
 
 def is_prod_appengine():
-    return ('APPENGINE_RUNTIME' in os.environ and
-            'Google App Engine/' in os.environ['SERVER_SOFTWARE'] and
-            not is_prod_appengine_mvms())
+    return not is_local_appengine()
```

Comment:
This now returns `True` if we're not in an AppEngine environment where it previously didn't. Maybe we should retain the `('APPENGINE_RUNTIME in os.environ' and ...` logic to this function?